core/src/main/scala/com/amadeus/dataio/pipes/kafka/streaming/KafkaOutput.scala

```diff
@@ -22,8 +21,8 @@ import scala.util.Try
  */
 case class KafkaOutput(
```

Comment:
This needs to be updated. The possibility to omit the topic altogether in the configuration is intentional in the Kafka-Spark API, and should not be prevented at framework level.

Fixes to do are in the _write_ and _createQueryName_ functions, as well as in the companion object.